drive/client.py

```diff
@@ -261,14 +261,14 @@ def list_files(self,
 
         q = _make_querystring(query_clauses)
 
-        return self._execute_file_request(self._files.list(q=q, pageSize=n))
+        return cast(List[File], self._execute_file_request(self._files.list(q=q, pageSize=n)))
 
     def update_file(self, file_id: str,
-                    remove_parents_ids=None,
-                    add_parents_ids=None,
+                    remove_parents_ids: Optional[Iterable[str]] = None,
+                    add_parents_ids: Optional[Iterable[str]] = None,
                     name: Optional[str] = None,
-                    media=None,
-                    force=False):
+                    media: Optional[MediaUpload] = None,
+                    force: bool = False) -> Optional[File]:
```

Comment:
`: bool` is not needed because `False` is already a `bool`. I think `Optional[File]` is redundant because Mypy can already see that the function returns either `None` either `cast(File, ...)`

Reply:
See above: https://github.com/bfontaine/drive/pull/13#discussion_r2015341970